scst-policy/configuring.md

```diff
@@ -51,25 +51,30 @@ Glob matches against images using semantics similar to golang filepaths. A `**`
   `myproject/nginx` instead of `index.docker.io/myproject/nginx`
 
 - If the image is specified without multiple path elements (not separated by
-  `/`), then library is defaulted. For example, specifying busybox causes library/busybox. And combined with earlier, causes a match being made
-  against index.docker.io/library/busybox.
+  `/`), then `library` is defaulted. For example, specifying `busybox`, the
+  resultant glob will default to `library/busybox`. And combined
+  with the previous defaulting, the resultant glob will be
+  `index.docker.io/library/busybox`.
 
 A sample of a ClusterImagePolicy which matches against all images using glob:
 
-```console
+```yaml
 apiVersion: policy.sigstore.dev/v1beta1
 kind: ClusterImagePolicy
 metadata:
   name: image-policy
 spec:
   images:
-  - glob: "*"
+  - glob: "**"
 ```
 
 ### <a id="cip-authorities"></a> `authorities`
 
-Authorities listed in the `authorities` block of the ClusterImagePolicy can be `key` specifications. Each `key` authority can contain a raw public key, a
-`secretRef` or a `kms` path.
+Authorities listed in the `authorities` block of the ClusterImagePolicy can be
+`key` or `keyless` specifications.
+
+Each `key` authority can contain a raw ECDSA public key, a `secretRef`, or a
```

Comment:
I am not sure if "raw" is the right term. For me raw is more like the bytes of the key. Maybe "PEM" or "PEM-encoded" is more appropriate.

Reply:
updated to PEM